### ISSUE TITLE
Could you release pretrained JDnDmSR without transfer learning

### ISSUE THREAD

**Issue by guochengqian** [xingwz/End-to-End-JDNDMSR#4]
Dear authors, 

thank you for the interesting work. I personally believe if you could release the pretrained model w/o transfer learning, it will benefit more to the community. Hope you can consider this suggestion.

**Comment by xingwz** [comment]
Sorry, we only have a specific jdndmsr model w/o transfer learning for noise level 10 and scale factor 2. And it is trained with MSE loss function.
The supported jdndmsr+.h5 is trained with MAE loss and transfer learning from the pertained jdmsr model (models/jdmsr.h5).

**Comment by guochengqian** [comment]
Hi @xingwz  thank you for your reply
